tracee-ebpf/Makefile

```diff
@@ -124,6 +124,10 @@ test: $(DOCKER_BUILDER)
 	$(call docker_builder_make,$@)
 endif
 
+.PHONY: test-integration
+test-integration:
```

Comment:
missing prerequisite (the binary)

Reply:
thanks updated https://github.com/aquasecurity/tracee/pull/606/commits/4ed2477e3cb3b7ecb1aa9f7f8ccb8532b73f28cc